lib/packages/docutils/rst.nim

```diff
@@ -1675,19 +1675,33 @@ proc parseMarkdownCodeblockFields(p: var RstParser): PRstNode =
       field.add(fieldBody)
       result.add(field)
 
+proc mayLoadFile(p: RstParser, result: var PRstNode) =
+  var filename = strip(getFieldValue(result, "file"),
+                       chars = Whitespace + {'"'})
+  if filename != "":
+    if roSandboxDisabled notin p.s.options:
+      let tok = p.tok[p.idx-2]
+      rstMessage(p, meSandboxedDirective, "file", tok.line, tok.col)
```

Comment:
If the sandbox is enabled, shouldn't we `return` here too? The idea is to not expose file contents from a server.

Reply:
For `me*` errors `rstMessage` throws an exception, there is no need to add extra returns (though we can).
Note that this code was not changed — it was just moved out to the separate proc from RST code blocks (well, I added `"` to chars for `strip` but not relevant here).
And it's checked by a new testcase in `trstgen.nim`.

Reply:
I noticed it was pre-existing but I asked anyway.